app/uk/gov/hmrc/gform/submission/DestinationAuditAlgebra.scala

```diff
@@ -18,25 +18,33 @@ package uk.gov.hmrc.gform.submission
 import java.util.UUID
 
 import cats.Applicative
-import cats.syntax.applicative._
-import cats.syntax.eq._
 import cats.instances.future._
 import cats.instances.string._
+import cats.syntax.applicative._
+import cats.syntax.eq._
 import org.joda.time.LocalDateTime
+import play.api.libs.json.Reads._
 import play.api.libs.json._
 import uk.gov.hmrc.gform.core.{ FOpt, fromFutureA }
 import uk.gov.hmrc.gform.form.FormAlgebra
 import uk.gov.hmrc.gform.repo.Repo
-import uk.gov.hmrc.gform.sharedmodel.UserId
+import uk.gov.hmrc.gform.sharedmodel.{ UserId, ValueClassFormat }
 import uk.gov.hmrc.gform.sharedmodel.form._
-import uk.gov.hmrc.gform.sharedmodel.formtemplate.{ FormComponentId, FormTemplateId }
 import uk.gov.hmrc.gform.sharedmodel.formtemplate.destinations.{ Destination, DestinationId }
+import uk.gov.hmrc.gform.sharedmodel.formtemplate.{ FormComponentId, FormTemplateId }
 import uk.gov.hmrc.http.HeaderCarrier
-import uk.gov.hmrc.mongo.json.ReactiveMongoFormats.localDateTimeWrite
+import uk.gov.hmrc.mongo.json.ReactiveMongoFormats._
 
-import scala.concurrent.ExecutionContext
+import scala.concurrent.ExecutionContext // Combinator syntax
```

Comment:
`// Combinator syntax` ?

Reply:
Weird. Not sure what happened there. Removed.